examples/02-HFSS/HFSS_eigenmode.py

```diff
@@ -0,0 +1,147 @@
+"""
+HFSS: Eigenmode Filter
+--------------------
+This example shows how you can use PyAEDT to automate eigenmode solver in HFSS.
+Eigenmode analysis can be applied to open, radiating structures
+using an absorbing boundary condition. This type of analysis can be very helpful
+to determine the resonant frequency of a geometry or an antenna and can be used to refine
+the mesh at the resonance, even when the resonant frequency of the antenna is not known.
+
+The challenge posed by this method is to identify and filter the non-physical modes
+resulting from reflection from boundaries of the main domain.
+Since the Eigenmode solver sorts by frequency and does not filter on the
+quality factor, these virtual modes will be present when the eigenmode approach is
+applied to nominally open structures.
+When looking for resonant modes over a wide frequency range for nominally
+enclosed structures, several iterations may be required as the minimum frequency
+is determined manually and simulations shall re-run until the complete frequency range is covered
+and all the important physical modes are calculated.
```

Comment:
```suggestion
Because the Eigenmode solver sorts by frequency and does not filter on the
quality factor, these virtual modes are present when the eigenmode approach is
applied to nominally open structures.
When looking for resonant modes over a wide frequency range for nominally
enclosed structures, several iterations may be required because the minimum frequency
is determined manually and simulations re-run until the complete frequency range is covered
and all important physical modes are calculated.
```